next/src/middleware.ts

```diff
@@ -60,6 +136,12 @@ const authMiddleware = (request: NextRequest, ctx: AppRouteHandlerFnContext) =>
       }
     }
 
+    // Check if there are redirects from Drupal for this request:
+    const redirectResponse = await handleDrupalRedirects(request);
+    if (redirectResponse) {
```

Comment:
I added this here, after the checks for auth pages, and before the middleware for next-intl 

(not sure this is great)